### ISSUE TITLE
Install Issues for MacOS Monterey 12.4

### ISSUE THREAD

**Issue by AgileEngineeringLLC** [righthalfplane/SdrGlut#30]
I get to the second step of the install, and get the following error. Any suggestions? Looks to be an X11 issue, and maybe Monterey is using XQuartz instead of X11 maybe?

(base) Dons-MacBook-Pro:~ donkelly$ brew tap dholm/homebrew-sdr
==> Tapping dholm/sdr
Cloning into '/usr/local/Homebrew/Library/Taps/dholm/homebrew-sdr'...
remote: Enumerating objects: 441, done.
remote: Total 441 (delta 0), reused 0 (delta 0), pack-reused 441
Receiving objects: 100% (441/441), 120.77 KiB | 1.49 MiB/s, done.
Resolving deltas: 100% (269/269), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/dholm/homebrew-sdr/multimon-ng.rb
multimon-ng: Unsupported special dependency :x11
Error: Cannot tap dholm/sdr: invalid syntax in tap!
(base) Dons-MacBook-Pro:~ donkelly$

**Comment by righthalfplane** [comment]
It looks like the old brew instruction are broke - I will have to update them.

If you just want to install -

https://github.com/righthalfplane/SdrGlut/releases/tag/v1.30

It has the prebuilt exe for MacOS Sierra, MacOS High Sierra, MacOS Mojave, MacOS Catalina and MacOS Big Sur and a native mode version for Apple-M1 Machines. Download SdrGlut-986-Install.dmg or SdrGlut-986-M1-m-Install.dmg and install SdrGlut by dragging it to the Applications folder. On MacOS, xquartz needs to be installed (https://www.xquartz.org/releases/). If you are Using a SDRPlay device, SDRplay_RSP_API-MacOSX-3.07.3 needs to be installed. You can get it from -

https://www.sdrplay.com/downloads/

**Comment by righthalfplane** [comment]
I have fixed the brew install instructions - The apple M1 machines need a few changes and a new Makefile.

**Comment by WAD0verfl0w** [comment]
Eh mate can you add the new M1 make file referenced in the instructions?

**Comment by righthalfplane** [comment]
Yes, Sorry about that - I put it out there a few minutes ago. I aways seem to forget something.
